pkgs/top-level/all-packages.nix

```diff
@@ -2736,9 +2736,10 @@ with pkgs;
 
   pcsxr = callPackage ../applications/emulators/pcsxr { };
 
-  ppsspp = callPackage ../applications/emulators/ppsspp { };
-
-  ppsspp-sdl = ppsspp;
+  ppsspp-sdl = ppsspp.override {
+    enableQt = false;
+    enableVulkan = true;
```

Comment:
This is the default already. Am I missing something?

Reply:
No. I am merely being explicit.